posts/2020/04/Spyder-4.md

```diff
@@ -0,0 +1,50 @@
+<!--
+.. title: Thanking the people behind Spyder 4
+.. slug: thanking-the-people-behind-spyder-4
+.. date: 2020-03-05 14:00:00 UTC-05:00
```

Comment:
Do you want to fix this?

Reply:
Sure, but @rgommers will take care of that before merging the PR.